package.json

```diff
@@ -41,7 +41,7 @@
   "peerDependencies": {
     "cldr-data": ">=25",
     "globalize": "^1.1.0-rc.5 <1.3.0",
-    "webpack": "^2.5.1"
+    "webpack": "^2.2.0-rc"
```

Comment:
It would. The user would have more options to manage their webpack version without GWP complaining about bad peer dependencies. Unless a specific webpack version is specified, running `yarn add "webpack@^2.2.0-rc"` will install 2.6.1